geoips/sector_utils/estimate_area_extent.py

```diff
@@ -298,7 +302,7 @@ def esitmate_area_from_center(lat_0, lon_0, height, width, resolution):
         estimated_extent = estimate_area_extent(
             ARGS.min_lat, ARGS.min_lon, ARGS.max_lat, ARGS.max_lon, ARGS.resolution
         )
-    except:
+    except argparse.ArgumentError:
```

Comment:
Is this the correct error to catch?

Reply:
I believe so, I did research and this seemed like the most likely one.